Tests.Integration.Presentation.Web/Interfaces/ItInterfacesTest.cs

```diff
@@ -151,6 +152,30 @@ public async Task Cannot_Set_Exposing_System(OrganizationRole role)
             Assert.Equal(HttpStatusCode.Forbidden, result.StatusCode);
         }
 
+        [Fact]
+        public async Task Can_Get_OrganizationName_Of_Organizations_Which_Use_The_Interface()
+        {
+            //Arrange
+            var organization = await OrganizationHelper.CreateOrganizationAsync(TestEnvironment.DefaultOrganizationId, A<string>(), string.Join("", Many<int>(8).Select(x => Math.Abs(x) % 9)), A<OrganizationTypeKeys>(), AccessModifier.Public);
+            var itInterface = await InterfaceHelper.CreateInterface(InterfaceHelper.CreateInterfaceDto(A<string>(), A<string>(), organization.Id, AccessModifier.Public));
+            var exhibitingSystem = await ItSystemHelper.CreateItSystemInOrganizationAsync(A<string>(), organization.Id, AccessModifier.Public);
+            var exhibitingSystemUsage = await ItSystemHelper.TakeIntoUseAsync(exhibitingSystem.Id, organization.Id);
+            var exhibit = await InterfaceExhibitHelper.CreateExhibit(exhibitingSystem.Id, itInterface.Id);
+
+            var usingSystem = await ItSystemHelper.CreateItSystemInOrganizationAsync(A<string>(), organization.Id, AccessModifier.Public);
+            var usingSystemUsage = await ItSystemHelper.TakeIntoUseAsync(usingSystem.Id, organization.Id);
```

Comment:
Hvis du nu laver en unit test af den property så kan du også få alle de der cases med hvor der er flere relationer (distinct) samt hvor der er navnesammenfald på forskellige organisationer og duplikaterne skal med dvs. group by (id, name).distinct().select(name).tolist

Reply:
Min pointe er at den her test er fin, men du har også brug for en unit test som kommer bredt omkring uden at det behøves at være en integrationstest, som er fin som den er her. De tjener forskellige formål